hypershift-operator/controllers/nodepool/kubevirt/kubevirt.go

```diff
@@ -147,65 +159,50 @@ func virtualMachineTemplateBase(image string, kvPlatform *hyperv1.KubevirtNodePo
 		},
 	}
 
+	if kvPlatform.RootVolume != nil {
+		if kvPlatform.RootVolume.Persistent != nil && kvPlatform.RootVolume.Persistent.Size != nil {
+			volumeSize = *kvPlatform.RootVolume.Persistent.Size
+		}
+
+		if kvPlatform.RootVolume.Persistent != nil {
+			storageClass = kvPlatform.RootVolume.Persistent.StorageClass
+
+			for _, am := range kvPlatform.RootVolume.Persistent.AccessModes {
+				accessModes = append(accessModes, corev1.PersistentVolumeAccessMode(am))
+			}
+
+			if len(accessModes) == 0 {
+				accessModes = append(accessModes, corev1.ReadWriteOnce)
+			}
```

Comment:
I don't think this is accurate. 

The accessModes slice should be empty if none are provided by the user. This is how the cdi storage spec works.